src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_ase_commands.py

```diff
@@ -0,0 +1,164 @@
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, record_only
+import json
+import os
+
+vault_ase_reg = 'ase-rsv-ccy'
+vault_ase_restore = 'ase-rsv-grs'
+reg_vm_name_ase = 'VMAppContainer;Compute;ase-rg-ccy;ase-ccy-vm2'
+res_vm_name_ase = 'VMAppContainer;Compute;ase-rg-ccy;ase-ccy-vm5'
+reg_vm_friendly_name = 'ase-ccy-vm2'
+res_vm_friendly_name = 'ase-ccy-vm5'
+vm5_friendly_name = 'ase-ccy-vm5'
+rg_ase = 'ase-rg-ccy'
+item_friendly_name = 'asetestdb1'
+reg_backup_policy = 'DailyPolicy-m9aya1dh'
+backup_item_name_db1 = 'SAPAseDatabase;ab4;asetestdb1'
+backup_item_name_db2 = 'SAPAseDatabase;ab4;asetestdb2'
+backup_item_name_db3 = 'SAPAseDatabase;ab4;asetestdb3'
+backup_item_name_db3_friendly_name = 'asetestdb3'
+reg_vm_id = '/subscriptions/38304e13-357e-405e-9e9a-220351dcce8c/resourceGroups/ase-rg-ccy/providers/Microsoft.Compute/virtualMachines/ase-ccy-vm2'
+
+# reg_vm_id = '/subscriptions/38304e13-357e-405e-9e9a-220351dcce8c/resourceGroups/ASE-RG-CCY/providers/Microsoft.Compute/virtualMachines/ase-ccy-vm2'
+# sub_ase = '38304e13-357e-405e-9e9a-220351dcce8c'
+# server_friendly_name = 'ase-ccy-vm2'
+# item_name = 'SAPAseDatabase;ab4;asetestdb2'
+
+
+class ASEBackupTests(ScenarioTest, unittest.TestCase):
+
+    def test_backup_wl_ase_container(self):
```

Comment:
rename test to be slightly more descriptive, looks good otherwise.

Reply:
Could be `test_backup_wl_ase_container_policy_operations`. Can remove the policy-only test that is getting skipped anyway later.